### ISSUE TITLE
Does this library already contain support for Kafka Streams introduced in Kafka 0.10

### ISSUE THREAD

**Issue by marcofranssen** [Jroland/kafka-net#105]
Below article explains very well what Kafka Streams is about.

https://timothyrenner.github.io/engineering/2016/08/11/kafka-streams-not-looking-at-facebook.html

Does this library already support them?
If not are there any plans to add it on the roadmap?

**Comment by javierholguera** [comment]
No, this library can't contain anything like that because Kafka Streams is a technology built on top of Java Official Kafka Client and only works on the Java Virtual Machine.
